test/workers/send_msg_of_user_worker_test.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
Dont think we need this file anymore, right?

Reply:
Yes, will delete it